go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/go-logr/logr v1.2.0
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.17.0
+	go.uber.org/zap v1.19.1
```

Comment:
this change doesn't make sense in the context of this PR?

Reply:
CI complains about the modified go.mod as well, so something is wrong, either the file, or CI. Check the make targets, order matters sometimes 🤷🏼‍♂️ (e.g. make generate and go fmt / goimports sometimes have slightly different output with import aliases)

Reply:
For me `go mod tidy` did the trick and reveled the changes

Reply:
I locally use Golang version 1.20. Maybe it matters

Reply:
It matters, always use the same (x.y at least) version locally.
I use https://github.com/travis-ci/gimme for quickly switching versions.

Reply:
Nice, I will give it a try. I will add the above change when I will update FAR to Golang v1.20 👍🏻